cassandra/src/main/java/org/apache/calcite/adapter/cassandra/CassandraSchema.java

```diff
@@ -140,7 +140,7 @@ public CassandraSchema(String host, int port, String keyspace, String username,
     this.parentSchema = parentSchema;
     this.name = name;
 
-    this.hook = Hook.TRIMMED.add(node -> {
+    this.hook = Hook.TRIMMED.addThread(node -> {
       CassandraSchema.this.addMaterializedViews();
     });
```

Comment:
Fix the compile warnings.

Reply:
@vlsi Can you help to see this problem ? when i try to replace the `add` with `addThread`, the jdk8 tests failed because of the concurrent modification exception.

Reply:
I guess @michaelmior could clarify better.

I've no idea what this does and I have very little understanding of how Calcite adapter works.
What I know is `Hook#add` adds a static (cross-thread) hook, which typically interferes when concurrent execution is used (e.g. when tests are executed concurrently).